e2e/config.js

```diff
@@ -64,6 +64,7 @@ module.exports = {
     amendOtherProceedings: 'Other proceedings',
     amendAttendingHearing: 'Attending the hearing',
     addHearingBookingDetails: 'Add hearing details',
+    provideStatementOfService: 'Add statement of service (c9)',
```

Comment:
I think it should be addStatmentOfServiceEventPage

Reply:
👍 changed